cmd/kube-scheduler/app/server.go

```diff
@@ -128,6 +128,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.Int32Var(&o.config.HardPodAffinitySymmetricWeight, "hard-pod-affinity-symmetric-weight", o.config.HardPodAffinitySymmetricWeight,
 		"RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule corresponding "+
 			"to every RequiredDuringScheduling affinity rule. --hard-pod-affinity-symmetric-weight represents the weight of implicit PreferredDuringScheduling affinity rule.")
+	fs.Int32Var(&o.config.NumberOfWorkQueueParallel, "number-of-workqueue-parallel", o.config.NumberOfWorkQueueParallel,
+		"NumberOfWorkQueueParallel to use when you need set the number of workqueue parallel.")
```

Comment:
`Number of workers running in parallel to find the nodes that fit the pods.`

WDYT?

Reply:
thank you so much for your suggestion.